cupyx/profiler/_time.py

```diff
@@ -112,7 +112,8 @@ def f(a, b):
             the timing test. If not given, the current device is used.
 
     Returns:
-        :class:`_PerfCaseResult`: an object collecting all test results.
+        :class:`~cupyx.profiler._time._PerfCaseResult`:
```

Comment:
If we are documenting this, should we remove the underscore?

Reply:
It's a good question... For now I am taking the view that the objects which 
- users should not construct themselves, but 
- can be returned by public APIs 

should keep the underscore prefix. This unifies the treatment for the cooperative group APIs.

Reply:
Seems reasonable!